x-pack/plugins/apm/common/elasticsearch_fieldnames.ts

```diff
@@ -114,6 +114,7 @@ export const LABEL_NAME = 'labels.name';
 
 export const HOST = 'host';
 export const HOST_NAME = 'host.hostname';
+export const HOSTNAME = 'host.name';
```

Comment:
Do we know why we use both fields? From ecs description they sounds interchangable.

Reply:
I think there's a subtle difference:

`host.name`

> Name of the host. It can contain what hostname returns on Unix systems, the fully qualified domain name, or a name specified by the user. The sender decides which value to use.

`host.hostname`

> Hostname of the host. It normally contains what the hostname command returns on the host machine.

My understanding is `host.hostname` is more "stable", as in it'll always mirror what a command like `hostname` returns, whereas `host.name` can be configured.

@jasonrhodes Do you have any more background on why logs / metrics use `host.name` and APM uses `host.hostname`, there are really old conversations on Slack about this, but they don't really say *why*. But for sure metrics leans on `host.name` for metrics uses, and logs uses `host.name` for "logs in context".

Reply:
Sorry, I missed your other comment when I wrote this question. I think we can just leave it for now.